devel/termcolor/Portfile

```diff
@@ -1,17 +1,20 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           cmake 1.1
+PortGroup           github  1.0
+PortGroup           cmake   1.1
 
-github.setup        ikalnytskyi termcolor 2.0.0 v
+github.setup        ikalnytskyi termcolor 2.1.0 v
+revision            0
 categories          devel
-platforms           darwin
 license             BSD
 maintainers         {l2dy @l2dy} openmaintainer
-description         a header-only C++ library for printing colored messages to the terminal
+
+description         \
+    a header-only C++ library for printing colored messages to the terminal
```

Comment:
Can you explain why this has to be reformatted?

Reply:
It was to have the description text stay within the 80-column boundary.  No big deal, I've restored it to how it was previously.